tests/api.c

```diff
@@ -20700,6 +20735,8 @@ static int test_wc_ed448_import_private_key (void)
                     "Ed448PublicKeyUnitTest.................................\n";
     word32      privKeySz = sizeof(privKey);
     word32      pubKeySz = sizeof(pubKey);
+    byte        bothKeys[sizeof(privKey) + sizeof(pubKey)];
```

Comment:
Same, should these be wrapped in `#ifdef HAVE_ED448_KEY_EXPORT`?

Reply:
done